pom.xml

```diff
@@ -261,82 +268,20 @@
         </executions>
       </plugin>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-antrun-plugin</artifactId>
-        <version>1.7</version>
+        <groupId>co.cask</groupId>
+        <artifactId>cdap-maven-plugin</artifactId>
+        <version>1.0.1-SNAPSHOT</version>
+        <configuration>
+          <cdapArtifacts>
+            <parent>system:cdap-data-pipeline[4.3.0-SNAPSHOT,9.0.0-SNAPSHOT)</parent>
```

Comment:
we probably shouldn't make this go up to 9.0.0. Maybe have 5.0.0-SNAPSHOT as the upper bound so it doesn't cross a major version.